next/13/replace-next-head-repomod/index.ts

```diff
@@ -0,0 +1,264 @@
+import tsmorph, {
+	Identifier,
+	JsxOpeningElement,
+	JsxSelfClosingElement,
+	ModuleKind,
+	Node,
+	SourceFile,
+} from 'ts-morph';
+
+import type {
+	Repomod,
+	UnifiedFileSystem,
+} from '@intuita-inc/repomod-engine-api';
+import type { fromMarkdown } from 'mdast-util-from-markdown';
+import type { visit } from 'unist-util-visit';
+import {
```

Comment:
That import won't work. We should copy the functions and types directly into this file.